### ISSUE TITLE
Link vertical axes

### ISSUE THREAD

**Issue by MaximMoinat** [rnugraha/react-covid19-viz#2]
When comparing plots from different countries, it could be nice be able to toggle to sharing the same range on the vertical axes.

In the example below, the UAB chart is actually a magnitude smaller than GB. 

![image](https://user-images.githubusercontent.com/17825660/80988600-031e9180-8e34-11ea-923a-7c5533a39f47.png)

**Comment by rnugraha** [comment]
Yep I realised this as well. Some countries has completely huge difference and the curve can be almost gone. But maybe will work on the logarithmic feature first.
